transformers/shared/constants/method.go

```diff
@@ -102,6 +102,9 @@ func catClawMethod() string     { return getSolidityFunctionSignature(Cat110ABI(
 func catFileBoxMethod() string {
 	return getOverloadedFunctionSignature(Cat110ABI(), "file", []string{"bytes32", "uint256"})
 }
+func catFileChopDunkMethod() string {
```

Comment:
Seems like we could just rename the `catFileChopLumpMethod()` to `catFileChopLumpDunkMethod()`?